### PR TITLE
Contributing ordereddict Python package

### PR DESCRIPTION
**Pull request by tloizou**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/o/ordereddict/ordereddict-1.1-goolf-1.4.10-Python-2.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+version = '1.1'
+
+homepage = 'https://pypi.python.org/pypi/ordereddict'
+description = """Drop-in substitute for Py2.7's new collections.OrderedDict. 
+The recipe has big-oh performance that matches regular dictionaries (amortized 
+O(1) insertion/deletion/lookup and O(n) iteration/repr/copy/equality_testing)."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://pypi.python.org/packages/source/o/%s/' % name.lower()]
```
Comment (boegel) [inline easybuild/easyconfigs/o/ordereddict/ordereddict-1.1-goolf-1.4.10-Python-2.7.3.eb]: please use `%(namelower)s`

File: easybuild/easyconfigs/o/ordereddict/ordereddict-1.1-goolf-1.4.10-Python-2.7.3.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+pythonversion = '2.7.3'
+pythonshortversion = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [(python, pythonversion)]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['.'],
```
Comment (boegel) [inline easybuild/easyconfigs/o/ordereddict/ordereddict-1.1-goolf-1.4.10-Python-2.7.3.eb]: please include a proper sanity check?

Comment (boegel) [inline easybuild/easyconfigs/o/ordereddict/ordereddict-1.1-goolf-1.4.10-Python-2.7.3.eb]: or, just drop `sanity_check_paths`; for stuff installed via `PythonPackage`, EasyBuild already does an `import` test, which should be good enough?

**Comment by tloizou** [comment]
@boegel: Changes made. Please test.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/eb53f17187687a09fe7b for a full test report.

**Comment by boegel** [comment]
Looking good now, thanks @tloizou!

**Pull request merged**